src/proposals/management/commands/export_proposals_create_time.py

```diff
@@ -0,0 +1,38 @@
+import json
+import pytz
+
+from datetime import datetime
+from django.utils import timezone
+from proposals.models import TalkProposal, TutorialProposal
+from django.core.management.base import BaseCommand
+
+PROPOSAL_TYPE_MAPPING = {
+    'talk': TalkProposal, 'tutorial': TutorialProposal
+}
+
+class Command(BaseCommand):
+    help = "export proposal's create time(default UTC+8) as JSON"
+
+    def handle(self, *args, **options):
+        self.export_proposals_create_time()
+
+    def export_proposals_create_time(self):
+        joint_proposals = []
+        for ptype, Proposal in PROPOSAL_TYPE_MAPPING.items():
+            for p in Proposal.objects.all():
+                timezone.activate(pytz.timezone('Asia/Taipei'))
+                # ^you can edit 'Asia/Taipei to other area.'
+                current_tz = timezone.get_current_timezone()
+                time = p.created_at
+                time = current_tz.normalize(time.astimezone(current_tz))
+                time = datetime.strftime(time, '%Y-%m-%d %H:%M:%S')
```

Comment:
I think you could do something like
```
timezone.localtime(p.created_at).strftime(...)
```

According to https://docs.djangoproject.com/en/3.0/topics/i18n/timezones/#usage (Usage 2.)
It will use the timezone set in django project settings.

But this will also do, maybe you can try this next time :D